File: src/weaponsprocurement/gui/StockReviewFilterGroup.java
package weaponsprocurement.gui;

enum StockReviewFilterGroup {
    SIZE("Size"),
    TYPE("Type"),
    DAMAGE("Damage");

    private final String label;

    StockReviewFilterGroup(String label) {
        this.label = label;
    }

    String getLabel() {
        return label;
    }
}

File: src/main/kotlin/weaponsprocurement/ui/stockreview/state/StockReviewFilterGroup.kt
package weaponsprocurement.ui.stockreview.state

enum class StockReviewFilterGroup(
    val label: String,
    val weaponOnly: Boolean,
) {
    SIZE("Size", true),
    TYPE("Type", true),
    DAMAGE("Damage", true),
    AVAILABILITY("Availability", false),
    RARITY("Rarity", false),
}
